src/applications/vaos/new-appointment/components/CommunityCareProviderSelectionPage/ProviderSelectionField.jsx

```diff
@@ -340,17 +358,19 @@ function ProviderSelectionField({
                 </button>
               </>
             )}
-            <button
-              type="button"
-              className="vaos-appts__cancel-btn va-button-link vads-u-margin--0 vads-u-flex--0"
-              onClick={() => {
-                setProvidersListLength(INITIAL_PROVIDER_DISPLAY_COUNT);
-                setShowProvidersList(false);
-              }}
-              aria-label="Cancel choosing a provider"
-            >
-              Cancel
-            </button>
+            {!providersListEmpty && (
```

Comment:
It might be good to keep the cancel button. You might have no providers when searching from current location and some providers when searching from residential address, which means you could get in a situation where you chose a provider, did another search, and changed the location.